bin/start_hn2.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# this is shell script to start HydraNorth2 running in docker container
+# Maintainer: strilets@ualberta.ca
+
+#make sure that database has been created
+while [ ! -e /db/${HYDRANORTH_DB:-hydranorth2dev} ]
+do
+    sleep 2
+done
+
+cd /app
+
+#check if db has been migrated
+if [ ! -e /db/initialized ]; then
+    sleep 10 && rake db:migrate && touch /db/initialized;
```

Comment:
same question as @murny mentioned on the didocker-hydranorth2 repo, is the check here needed? db:migrate can be run as many times as you need.